### PR TITLE
Improve wording tooltip Traffic & SEO > Schema of URLs

### LINKED ISSUE #27476: Improve wording tooltip Traffic & SEO > Schema of URLs
### Prerequisites

- [X] I understand and accept the project's [code of conduct](https://github.com/PrestaShop/PrestaShop/blob/develop/CODE_OF_CONDUCT.md)
- [X] I have already [searched in existing features request](https://github.com/PrestaShop/PrestaShop/issues?q=is%3Aissue+label%3AImprovement%2CFeature%2CDeveloper+Feature+) and found no previous suggestion of this feature.

### Is your feature request related to a problem?

Following [@MatShir's comment ](https://github.com/PrestaShop/PrestaShop/pull/27234#issuecomment-1011953702) I'd like to suggest a new wording for the tooltip in Traffic & SEO > Schema of URLs.

### Describe the solution you'd like

On Traffic & SEO > Schema of URLs, to add a fixed warning alert before the features (in Admin.Shopparameters.Notification):

'Before making any changes to the default pattern of your URLs, make sure to meet the following requirements:

- The Friendly URL feature must be enabled
- URL rewriting must be activated on your web server
- All mandatory keywords (*) must be included in the route

You should also comply with the syntax:

- Keywords must be in braces {keyword}
- All keywords must be separated
- As long as a keyword has an associated value, you can use the syntax {prepend:keyword:append} to add text before and after it'

### Alternatives you've considered

_No response_

### Additional context

_No response_

### Do you plan to work on this feature?

- [X] I'm willing to contribute a [formal specification](https://github.com/PrestaShop/prestashop-specs).
- [ ] I'm willing to provide any wireframes or design assets required for this feature.
- [ ] I'm willing to submit a Pull Request that implements this feature.
- [X] I'm willing to help verify that the implemented feature works as intended and produces no unintended side effects.

### PR DESCRIPTION
**Pull request by ghost**
| Questions         | Answers
| ----------------- | -------------------------------------------------------
| Branch?           | develop
| Description?      | Improve wording tooltip Traffic & SEO > Schema of URLs
| Type?             | improvement
| Category?         | BO 
| BC breaks?        | no
| Deprecations?     |  no
| Fixed ticket?     | Fixes https://github.com/PrestaShop/PrestaShop/issues/27476
| How to test?      | See issue
| Possible impacts? | Please indicate what parts of the software we need to check to make sure everything is alright.

Capture d’écran de 2022-01-28 11-54-45

---
This change is [Reviewable](https://reviewable.io/reviews/prestashop/prestashop/27510)

### CODE REVIEW

**Comment by Julievrz** [comment]
Hi @okom3pom and thanks!

Is it normal that the font size is bigger in the bullet points?

PS: Could we remove the title "Read more"?

**Review by Progi1984 — DISMISSED** [review]

**Comment by ghost** [comment]
@Julievrz

I found the way for the size but for << read more >>  I don't know... 

The bubble is closed with << Read More >>

Capture d’écran de 2022-02-01 20-41-12

@NeOMakinG Can you tell us more?

**Comment by ghost** [comment]
I found the way by removing alert-text

Capture d’écran de 2022-02-02 09-17-07

File: src/PrestaShopBundle/Resources/views/Admin/Configure/ShopParameters/TrafficSeo/Meta/Blocks/url_schema_configuration.html.twig [hunk 1]
```diff
@@ -44,6 +44,32 @@
     </h3>
     <div class="card-block row">
       <div class="card-text">
+        <div class="alert alert-warning" role="alert">
+            {{ 'Before making any changes to the default pattern of your URLs, make sure to meet the following requirements:'|trans({}, 'Admin.Shopparameters.Help') }}
+            <ul>
+              <li>
+                {{ 'The Friendly URL feature must be enabled'|trans({}, 'Admin.Shopparameters.Help') }}
```
Comment (Julievrz) [inline src/PrestaShopBundle/Resources/views/Admin/Configure/ShopParameters/TrafficSeo/Meta/Blocks/url_schema_configuration.html.twig]:
```suggestion
                {{ 'The Friendly URL feature must be enabled'|trans({}, 'Admin.Shopparameters.Notification') }}
```

File: src/PrestaShopBundle/Resources/views/Admin/Configure/ShopParameters/TrafficSeo/Meta/Blocks/url_schema_configuration.html.twig [hunk 2]
```diff
@@ -44,6 +44,32 @@
...
+              </li>
+              <li>
+                {{ 'URL rewriting must be activated on your web server'|trans({}, 'Admin.Shopparameters.Help') }}
```
Comment (Julievrz) [inline src/PrestaShopBundle/Resources/views/Admin/Configure/ShopParameters/TrafficSeo/Meta/Blocks/url_schema_configuration.html.twig]:
```suggestion
                {{ 'URL rewriting must be activated on your web server'|trans({}, 'Admin.Shopparameters.Notification') }}
```

File: src/PrestaShopBundle/Resources/views/Admin/Configure/ShopParameters/TrafficSeo/Meta/Blocks/url_schema_configuration.html.twig [hunk 3]
```diff
@@ -44,6 +44,32 @@
...
+              </li>
+              <li>
+                {{ 'All mandatory keywords (*) must be included in the route'|trans({}, 'Admin.Shopparameters.Help') }}
```
Comment (Julievrz) [inline src/PrestaShopBundle/Resources/views/Admin/Configure/ShopParameters/TrafficSeo/Meta/Blocks/url_schema_configuration.html.twig]:
```suggestion
                {{ 'All mandatory keywords (*) must be included in the route'|trans({}, 'Admin.Shopparameters.Notification') }}
```

File: src/PrestaShopBundle/Resources/views/Admin/Configure/ShopParameters/TrafficSeo/Meta/Blocks/url_schema_configuration.html.twig [hunk 4]
```diff
@@ -44,6 +44,32 @@
...
+              </li>
+            </ul>
+            {{ 'You should also comply with the syntax:'|trans({}, 'Admin.Shopparameters.Help') }}
```
Comment (Julievrz) [inline src/PrestaShopBundle/Resources/views/Admin/Configure/ShopParameters/TrafficSeo/Meta/Blocks/url_schema_configuration.html.twig]:
```suggestion
            {{ 'You should also comply with the syntax:'|trans({}, 'Admin.Shopparameters.Notification') }}
```

File: src/PrestaShopBundle/Resources/views/Admin/Configure/ShopParameters/TrafficSeo/Meta/Blocks/url_schema_configuration.html.twig [hunk 5]
```diff
@@ -44,6 +44,32 @@
...
+            <ul>
+              <li>
+                {{ 'Keywords must be in braces {keyword}'|trans({}, 'Admin.Shopparameters.Help') }}
```
Comment (Julievrz) [inline src/PrestaShopBundle/Resources/views/Admin/Configure/ShopParameters/TrafficSeo/Meta/Blocks/url_schema_configuration.html.twig]:
```suggestion
                {{ 'Keywords must be in braces {keyword}'|trans({}, 'Admin.Shopparameters.Notification') }}
```

File: src/PrestaShopBundle/Resources/views/Admin/Configure/ShopParameters/TrafficSeo/Meta/Blocks/url_schema_configuration.html.twig [hunk 6]
```diff
@@ -44,6 +44,32 @@
...
+              </li>
+              <li>
+                {{ 'All keywords must be separated'|trans({}, 'Admin.Shopparameters.Help') }}
```
Comment (Julievrz) [inline src/PrestaShopBundle/Resources/views/Admin/Configure/ShopParameters/TrafficSeo/Meta/Blocks/url_schema_configuration.html.twig]:
```suggestion
                {{ 'All keywords must be separated'|trans({}, 'Admin.Shopparameters.Notification') }}
```

File: src/PrestaShopBundle/Resources/views/Admin/Configure/ShopParameters/TrafficSeo/Meta/Blocks/url_schema_configuration.html.twig [hunk 7]
```diff
@@ -44,6 +44,32 @@
...
+              </li>
+              <li>
+                {{ 'As long as a keyword has an associated value, you can use the syntax {prepend:keyword:append} to add text before and after it'|trans({}, 'Admin.Shopparameters.Help') }}
```
Comment (Julievrz) [inline src/PrestaShopBundle/Resources/views/Admin/Configure/ShopParameters/TrafficSeo/Meta/Blocks/url_schema_configuration.html.twig]:
```suggestion
                {{ 'As long as a keyword has an associated value, you can use the syntax {prepend:keyword:append} to add text before and after it'|trans({}, 'Admin.Shopparameters.Notification') }}
```

**Review by Julievrz — COMMENTED** [review]
I made a mistake in the translation domain in the issue because I thought this would be a tooltip like before. But as it is now a warning message, the right domain is Admin.Shopparameters.Notification

Sorry 😅 and thank you 🤗

**Comment by ghost** [comment]
Done @Julievrz

**Review by matks — DISMISSED** [review]

**Review by atomiix — DISMISSED** [review]

**Comment by florine2623** [comment]
Hello @okom3pom ,

Thanks for the PR !

I'm having an issue on Traffic & SEO page when installing this PR:
Screenshot 2022-02-04 at 15 59 52

Could you check please ?
Thanks!

**Comment by florine2623** [comment]
Hello @okom3pom ,

Could you make sure that all checks have passed ?
Thanks!

**Comment by Progi1984** [comment]
@okom3pom Could you rebase your PR (conflicting file) ?

**Comment by ghost** [comment]
Rebase ok

**Review by atomiix — APPROVED** [review]

**Review by florine2623 — COMMENTED** [review]
@okom3pom ,
LGTM, thanks !

@PrestaShop/prestashop-core-developers ,
Can someone approve the modifications before I put label QA ✅ ?

Thanks!

**Review by saulaski — APPROVED** [review]

**Review by Progi1984 — APPROVED** [review]

**Comment by Progi1984** [comment]
@Julievrz Could you approve it, please ?

**Comment by Julievrz** [comment]
> @Julievrz Could you approve it, please ?

Yep, that was already approved. ☺️

**Comment by Progi1984** [comment]
@florine2623 It's good.

**Review by florine2623 — APPROVED** [review]
Hello all,

It is looking good to me ! QA ✅ 
Thanks!

**Comment by Progi1984** [comment]
Thanks @okom3pom & @florine2623

**Pull request merged**